includes/class-linkbacks-handler.php

```diff
@@ -15,10 +15,9 @@ class Linkbacks_Handler {
 	public static function init() {
 		// enhance linkbacks
 		add_filter( 'preprocess_comment', array( 'Linkbacks_Handler', 'enhance' ), 0, 1 );
-		add_filter( 'wp_update_comment_data', array( 'Linkbacks_Handler', 'enhance' ), 11, 3 );
-
-		// Updates Comment Meta if set in commentdata
-		add_action( 'edit_comment', array( 'Linkbacks_Handler', 'update_meta' ), 10, 2 );
+		// Since currently wp_update_comment_data filters out comment meta using alternate update method.
+		add_action( 'edit_webmention', array( 'Linkbacks_Handler', 'update' ), 10, 2 );
```

Comment:
this does not work, because it is an action and the `update` or `enhance` function does not safe data. I do not understand why you want to replace `wp_update_comment_data` with something else.